### DISCUSSION TITLE
seed in production env

### DISCUSSION THREAD

**Discussion by ranran2121** [prisma/prisma#16674] [Q&A]
Hi,

sorry for the question but I am not well experienced yet. I need help to solve the following scenario:
I need to populate a table of my DB and I created a seed file. According to the documentation, I understand that

1. I can run the seed manually (via the prisma db seed), which works great

2. or via the prisma migrate dev , which does NOT work if I do not have conflicts. This option works only if I first run prisma reset and then prisma migrate dev

Apart from this issue, I will then need to apply migration (and seed) in the QA and PROD env, too. And I cannot reset the DB of course . Any suggestion about

-why the prisma migrate dev is not running the seed?
-how to apply the seed in QA/PROD env?
Thank you

**Comment by nurul3101** [comment]
Hello @ranran2121 👋 

1) Does your seed command not run automatically after you select `yes` to the reset prompt when invoking `migrate dev` command? 

2) To run the `npx prisma db seed` command in QA/Prod environments, you would need to change the DATABASE_URL to point to QA/Production databases. You can have different env files for different environments. Here's a [guide](https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files) on how you can use multiple env files. Additionally, you can have different commands in your package.json which invokes seed commands for different environments by reading DATABASE_URL from different env files.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
